kibit/src/kibit/rules/collections.clj

```diff
@@ -20,5 +20,7 @@
   ;; set
   [(into #{} ?coll) (set ?coll)]
 
-  [(take ?n (repeatedly ?coll)) (repeatedly ?n ?coll)])
+  [(take ?n (repeatedly ?coll)) (repeatedly ?n ?coll)]
+  [(doall (map ?fn ?coll)) (run! ?fn ?coll)]
```

Comment:
Fair point. Updated

Would be good if kibit could somehow check if the resulting sequence is actually used. If the resulting sequence is ignored, it could be replaced with `run!`